docs/streamlit_configuration.md

```diff
@@ -226,6 +222,10 @@ token = ""
 
 [deprecation]
 
+# Set to false to disable the deprecation warning for the file uploader encoding.
+# Default: "True"
+showfileUploaderEncoding = "True"
```

Comment:
hm this should be expired. Let's not add this here. I'm assuming you just did a `streamlit config show`? If so we may need to revisit that expiration flag to see why it's showing up

Reply:
I just did a `streamlit config show`, yes. I'll remove.